src/averageAges.js

```diff
@@ -15,11 +15,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredPeople = people.filter(el =>
+    (Math.ceil(el.died / 100) === (century || Math.ceil(el.died / 100)))
+   && (el.sex !== 'f'));
```

Comment:
```suggestion
  const filteredPeople = people.filter(el => (
    (Math.ceil(el.died / 100) === (century || Math.ceil(el.died / 100)))
   && (el.sex !== 'f')));
```
If you move your shortened return statement in arrow function to a newline, use round brackets (p. 2 of checklist)